reactioncommerce/reaction
# [Prototype] Add Storybook support and some examples

Impact: **minor**  
Type: **docs**

## Issue

Documentation should live closer to home to make it easier to write docs while you code.

## Solution

Add Storybook with a few documentation examples.

This prototype has 3 different location for docs.

- ConfirmDialog as its story next to the implementation
![image](https://user-images.githubusercontent.com/42217/58738671-a7de5680-83bb-11e9-886e-3c4f54b6e01f.png)

- The register components package has a `docs/stories` folder as well as a `docs/developer` folder containing the stories, and markdown files.

![image](https://user-images.githubusercontent.com/42217/58738684-b9bff980-83bb-11e9-8790-fb7500e745ab.png)

- `/imports/stories`

![image](https://user-images.githubusercontent.com/42217/58738746-ea079800-83bb-11e9-87b1-79a8868197ba.png)


## Breaking changes

none

## Testing
1. Run in docker `docker-compose up storybook` or...
2. Run outside doctor `npm install && npm run storybook`
3. Go to http://localhost:3100 if you're not automatically taken there (usually because of docker)

See something like this:

![image](https://user-images.githubusercontent.com/42217/58738575-37373a00-83bb-11e9-89fe-0193ef070582.png)
